.changesets/turn-error-backend-off-by-default.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "minor"
+type: "change"
+---
+
+Turn error backend off by default
```

Comment:
```suggestion
Turn error backend off by default. This will prevent errors from being reported without much context. If you're missing errors after this change, set the `enable_error_backend` config option to `true`, and let us know what errors you're missing at [support@appsignal.com](mailto:support@appsignal.com).
```